chore(pwa): tidy up parent-child links (#39585)
mdn/content: files/en-us/web/progressive_web_apps/index.md

## How to

> These [PWA how-tos](/en-US/docs/Web/Progressive_web_apps/How_to) give concrete detailed instructions for how to implement specific PWA features.

Reviewer: ```suggestion
The [PWA how-tos](/en-US/docs/Web/Progressive_web_apps/How_to) provide detailed instructions on implementing specific PWA features.
```

## Tutorials

> Build a PWA from scratch using these [PWA Tutorials](/en-US/docs/Web/Progressive_web_apps/Tutorials) which walk through the steps from start to finish, explaining how the different features of the app are implemented along the way.

Reviewer: ```suggestion
Build a PWA from scratch using these [PWA Tutorials](/en-US/docs/Web/Progressive_web_apps/Tutorials), which walk through the steps from start to finish, explaining how the different features of the app are implemented along the way.
```

## Reference

> [PWA reference](/en-US/docs/Web/Progressive_web_apps/Reference) page lists all features documented on MDN that you use to build a PWA.

Reviewer: ```suggestion
Our [PWA reference](/en-US/docs/Web/Progressive_web_apps/Reference) lists all features documented on MDN that you'll need to build a PWA.
```
